arcgis-ios-sdk-samples/Edit data/Edit features with feature-linked annotation/EditFeaturesWithFeatureLinkedAnnotationViewController.swift

```diff
@@ -0,0 +1,253 @@
+        // Clear the selection and selected feature if "No" is selected.
+        alert.addAction(UIAlertAction(title: "Cancel", style: .cancel) { _ in
+            self.clearSelection()
+        })
+        alert.addAction(UIAlertAction(title: "Move", style: .default) { _ in
+            // Get the selected feature's geometry as a polyline nearest to the map point.
+            guard let polyline = selectedFeature.geometry as? AGSPolyline,
+                let selectedMapPoint = AGSGeometryEngine.projectGeometry(mapPoint, to: polyline.spatialReference!) as? AGSPoint,
+                let nearestVertex = AGSGeometryEngine.nearestVertex(in: polyline, to: selectedMapPoint) else { return }
+            let polylineBuilder = AGSPolylineBuilder(polyline: polyline)
+            // Get the part of the polyline nearest to the map point.
+            polylineBuilder.parts[nearestVertex.partIndex].removePoint(at: nearestVertex.partIndex)
+            // Add the map point as the new point on the polyline.
+            polylineBuilder.parts[nearestVertex.partIndex].addPoint(AGSGeometryEngine.projectGeometry(mapPoint, to: polylineBuilder.parts[nearestVertex.partIndex].spatialReference!) as! AGSPoint)
+            // Set the selected feature's geometry to the new polyline.
+            selectedFeature.geometry = polylineBuilder.toGeometry()
+            // Update the selected feature's feature table.
+            selectedFeature.featureTable?.update(selectedFeature) { [weak self] (error: Error?) in
+                guard let self = self else { return }
+                if let error = error {
+                    self.presentAlert(error: error)
+                } else {
+                    // Clear selection of polyline.
+                    self.clearSelection()
+                }
+            }
+        })
+        present(alert, animated: true)
+    }
+    
+    // Clear selection from all feature layers.
+    func clearSelection() {
+        selectedFeature = nil
+        mapView.map?.operationalLayers.forEach { layer in
+            if let featureLayer = layer as? AGSFeatureLayer {
+                featureLayer.clearSelection()
+            }
+        }
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Add the source code button item to the right of navigation bar.
+        (self.navigationItem.rightBarButtonItem as! SourceCodeBarButtonItem).filenames = ["EditFeaturesWithFeatureLinkedAnnotationViewController"]
+    }
+}
+
+// MARK: - AGSGeoViewTouchDelegate
+// Select the nearest feature or move the point or polyline vertex to the given screen point.
+extension EditFeaturesWithFeatureLinkedAnnotationViewController: AGSGeoViewTouchDelegate {
+    func geoView(_ geoView: AGSGeoView, didTapAtScreenPoint screenPoint: CGPoint, mapPoint: AGSPoint) {
+        if let identifyLayers = identifyLayers {
```

Comment:
The meaning of `identifyLayers` at the point of use is a little confusing. Based on the name, I would think that maybe it is an array of layers? Perhaps a better name would be `identifyOperation`.